### ISSUE TITLE
[iam-user] This object has no argument, nested block, or exported attribute named "password".

### ISSUE THREAD

**Issue by rblaine95** [terraform-aws-modules/terraform-aws-iam#213]
## Description

When using `"terraform-aws-modules/iam/aws//modules/iam-user"` with `create_iam_user_login_profile = false`, terraform fails with the following error:
```
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/iam_user/modules/iam-user/outputs.tf line 33, in output "iam_user_login_profile_password":
│   33:   value       = try(aws_iam_user_login_profile.this[0].password, "")
│ 
│ This object has no argument, nested block, or exported attribute named "password".
```

Rolling back to `4.16.0` seems to fix the issue.

## Versions

- Module version [Required]: `4.17.1` (`~> 4.17`)

- Terraform version:
```
Terraform v1.1.7
on darwin_arm64
```
- Provider version(s):
```
+ provider registry.terraform.io/hashicorp/archive v2.2.0
+ provider registry.terraform.io/hashicorp/aws v3.75.1
+ provider registry.terraform.io/hashicorp/kubernetes v2.9.0
+ provider registry.terraform.io/hashicorp/tls v3.1.0
```

## Reproduction Code [Required]

```hcl
terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 3.4"
    }
  }
  required_version = ">= 1.0"
}

provider "aws" {
  region = "eu-west-1"
}

module "iam_user" {
  source  = "terraform-aws-modules/iam/aws//modules/iam-user"
  version = "~> 4.17"

  name          = "example.iam-user"
  force_destroy = true

  create_iam_user_login_profile = false
}
```
```sh
terraform init
terraform plan --target=module.iam_user
```

## Expected behavior
IAM User should be created as expected without a login profile.

**Comment by rblaine95** [comment]
Looks like it's only available with AWS Provider v4
* v3.75.1 [Docs](https://registry.terraform.io/providers/hashicorp/aws/3.75.1/docs/resources/iam_user_login_profile)
* v4.1.0 [Docs](https://registry.terraform.io/providers/hashicorp/aws/4.1.0/docs/resources/iam_user_login_profile)

**Comment by RafPe** [comment]
But it has been added as there was setup done towards 4.x provider https://github.com/terraform-aws-modules/terraform-aws-iam/compare/v4.16.0...v4.17.1

**Comment by rblaine95** [comment]
Yes, in the issue above I'm using AWS Provider `v3.75.1` which doesn't have the `password` attribute.
If I upgrade to `>= v4.1.0` then the module executes exactly as expected. Unfortunately I'm not able to migrate to AWS `4.x` just yet.

It's odd that the `try()` block fails 🤔 
I'd think the `try()` block would try and just return `""` if there's an error 🤔

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.17.2 :tada:

**Comment by bryantbiggs** [comment]
`pleasedontfailmenow(tryharder(try(var.sensitive)))`
